config-templates/redhat/odf-local/4.14/storage-class.yaml

```diff
@@ -0,0 +1,139 @@
+apiVersion: v1
+kind: List
+items:
+  - apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+      name: sat-ocs-cephrbd-gold
+      labels:
+        app.kubernetes.io/name: "redhat-odf-local"
+        app.kubernetes.io/version: "v4.14.0"
```

Comment:
The image version in deployment file is `v4.14.1`. So do we want to keep version here as `v4.14.0` or `v4.14.1`?